pkg/catalog/manager/catalog_common.go

```diff
@@ -99,10 +104,11 @@ func (m *Manager) updateCatalogInfo(cmt *CatalogInfo, catalogType string, templa
 		}
 		v3.CatalogConditionRefreshed.Unknown(obj)
 		if templateName != "" {
-			v3.CatalogConditionRefreshed.Message(obj, fmt.Sprintf("syncing template %v", templateName))
+			v3.CatalogConditionRefreshed.Message(obj, fmt.Sprintf("syncing catalog %v", cmt.catalog.Name))
 		} else {
 			v3.CatalogConditionRefreshed.Message(obj, fmt.Sprintf(""))
 		}
+
```

Comment:
?

Reply:
I had to move the method that was calling catalog update outside the template loop, so it no longer updates for each template.  This means the sync message needs to be at a catalog level and not for each template.